cmd/sifnoded/cmd/root.go

```diff
@@ -49,7 +49,6 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		Use:   "sifnoded",
 		Short: "app Daemon (server)",
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
-			initClientCtx = client.ReadHomeFlag(initClientCtx, cmd)
```

Comment:
@Jedi2002 any reason why we are getting rid of this one?

Reply:
The ReadHomeFlag was removed by the SDK in this version. 
We are already setting the home directory to the default when initializing `initClientCtx` so this should not be a problem